### PR TITLE
Added node 5.1.0 definition

### PR DESCRIPTION
**Pull request by pongstr**

### CODE REVIEW

**Comment by jasonkarns** [comment]
Can you revert this commit (fc3411abc54c8b166e297622164c657201d87a97) from the pr?

**Comment by jasonkarns** [comment]
actually, there are a lot of other commits in this PR that aren't related to the 5.1.0 definition. Can you rebase and we'll reassess?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # node-build
 
-[![Build Status](https://travis-ci.org/OiNutter/node-build.png?branch=master)](https://travis-ci.org/OiNutter/node-build)
+[![Build Status](https://travis-ci.org/pongstr/node-build.svg?branch=master)](https://travis-ci.org/pongstr/node-build)
```
Comment (jasonkarns) [inline README.md]: switching to svg is a good idea, but not sure why you changed all the links to your username. We need to keep the links pointing to the main repo (all below, too)

**Comment by jasonkarns** [comment]
instead of cleaning up this PR, I just ran the scraper myself. thanks for the PR, though. next time, be sure you're running on latest master before running the scraper and the PR will be cleaner.

**Comment by pongstr** [comment]
Apologies, I overlooked this yesterday.

**Pull request closed without merging**